### PR TITLE
Added parameters to procedure coverage integration test

### PR DESCRIPTION
**Pull request by sfc-gh-astus**
### Pre-review checklist
   * [x] I've confirmed that instructions included in README.md are still correct after my changes in the codebase.
   * [ ] I've added new automated tests to verify correctness of my new code.
   * [x] I've confirmed that my changes are up-to-date with the target branch.
   * [ ] I've described my changes in the release notes.
   * [x] I've described my changes in the section below.

### Changes description
Added parameters for procedure in procedure coverage integration test
Procedure and function have unit tests

### CODE REVIEW

File: src/snowcli/utils.py [hunk 1]
```diff
@@ -570,7 +570,8 @@ def convert_resource_details_to_dict(function_details: SnowflakeCursor) -> dict:
 
 def generate_deploy_stage_name(identifier: str) -> str:
     return (
-        identifier.replace(
+        identifier.replace("()", "")
```
Comment (sfc-gh-pjob) [inline src/snowcli/utils.py:573]: 1. Why this change is required?
2. Do I see correctly that this change will affect also functions and procedures, not only procedure coverages? If yes, then the release notes should be updated.

File: tests_integration/test_procedure_coverage.py [hunk 2]
```diff
@@ -26,40 +26,46 @@ def test_procedure_coverage_flow(_test_steps):
...
     _test_steps.snowpark_init_should_initialize_files_with_default_content()
+    _test_steps.add_parameters_to_procedure("a: int, b: str")
     _test_steps.add_requirements_to_requirements_txt(["coverage"])
     _test_steps.requirements_file_should_contain_coverage()
     _test_steps.snowpark_package_should_zip_files()
 
+    parameters = "(a int, b string)"
     procedure_name = (
-        _test_steps.snowpark_create_with_coverage_wrapper_should_finish_succesfully()
+        _test_steps.snowpark_create_with_coverage_wrapper_should_finish_successfully(
```
Comment (sfc-gh-pjob) [inline tests_integration/test_procedure_coverage.py:36]: Do we know what will happen when an update will be with parameters `"(c int, d string)"`? Do we have any test case for such scenario?

File: src/snowcli/utils.py (hunk 1 above)
Comment (sfc-gh-astus) [reply src/snowcli/utils.py:573]: 1. In previous PR, I changed replace `(` to `_`, but later I found that there is second function doing almost the same thing (one of them is lowering result). When I changed replace in second function, tests with no parameters began to fail.

File: tests_integration/test_procedure_coverage.py (hunk 2 above)
Comment (sfc-gh-astus) [reply tests_integration/test_procedure_coverage.py:36]: I totally forgot about it

**Review by sfc-gh-turbaszek — APPROVED** [review]

File: tests_integration/test_procedure_coverage.py (hunk 2 above)
Comment (sfc-gh-pjob) [reply tests_integration/test_procedure_coverage.py:36]: We agreed to create a separate task for fixing this case.

**Review by sfc-gh-pjob — APPROVED** [review]
